### DISCUSSION TITLE
错误的类SQL指令也要写入文件吗？

### DISCUSSION THREAD

**Discussion by Laurel-Canyon** [liusy58/NJU_AdvancedProgramming_SP22#122] [Q&A] [answered]
PPT中提到“选择一个操作时，首先要生成一个指令”，那么如果用户选择的操作是错误的（如下架已经下架过的商品、封禁已经封禁过的用户等），这类也要写入吗...还是说能写入的只有能完全正确执行的指令？

**Comment by huanhuan6666** [comment]
同问，只有成功执行的SQL指令才记录到command.txt文件吗？还是生成的SQL指令不管有没有成功执行都记录到文件中

**Comment by RicoloveFeng** [comment, marked as answer]
> 只有成功执行的指令才被写入command.txt文件。
